app.js

```diff
@@ -118,20 +100,56 @@ export default class App extends Component {
     const copiedState = [...this.state.data]
     const formObject = {
       question: this.state.question,
-      answer: this.state.answer
+      answer: this.state.answer,
     }
     copiedState.push(formObject)
     this.setState({
       data: copiedState,
       question: '',
-      answer: ''
+      answer: '',
     })
     target.reset()
   }
 
+  handleDestroy(event) {
+    const { target } = event
+    const delIndex = parseInt(target.parentNode.parentNode.getAttribute('data-id'))
+    const copy = [...this.state.data]
+    copy.forEach((elem, index) => {
```

Comment:
If you already have the `delIndex`, you shouldn't need to loop through the `copy`, just call `splice` on the `copy`.